### DISCUSSION TITLE
blog/how-to-one-way-anova-by-hand/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#107] [General]
# blog/how-to-one-way-anova-by-hand/

Learn how to perform a one-way Analysis Of VAriance (ANOVA) by hand to compare a quantitative measure between three groups or more

https://statsandr.com/blog/how-to-one-way-anova-by-hand/

**Comment by alohaia** [comment]
Great post, it's helpful.

**Reply by AntoineSoetewey** [reply]
Glad you find it useful @alohaia!

**Comment by thiagonazario** [comment]
I really like this ANOVA by hand! Thanks!

**Reply by AntoineSoetewey** [reply]
You’re welcome @thiagonazario!

**Comment by nop9898** [comment]
Hi Antoine - thank you for this write-up, 

I think there's a small typo in your SSR calculation for group C. 

You have: 4 * ((18 - 26)^2) = 64 
But I think you mean 4 * ((18 - **22**)^2) = 64

(the 64 outcome value is correct)

**Reply by AntoineSoetewey** [reply]
Good catch @nop9898. It has been corrected, thanks!
